android-enaml/build.sh

```diff
@@ -8,44 +8,40 @@
 export HOSTPYTHON=$PYTHON
 source $BUILD_PREFIX/android/activate-ndk.sh
 
-if [ "$PY3K" == "1" ]; then
-    export PY_LIB_VER="3.10"
-else
-    export PY_LIB_VER="2.7"
-fi
-
-# Install in host python
-python setup.py develop
-
-# Write parse tables
-python $RECIPE_DIR/write_tables.py
-
 
 for ARCH in $ARCHS
 do
 
     # Setup compiler for arch and target_api
+  
     activate-ndk-clang $ARCH
+
     export CFLAGS="$CFLAGS -O3 -I$APP_ROOT/include/python$PY_LIB_VER"
-    export LDFLAGS="$LDFLAGS -lpython$PY_LIB_VER"
+    export LDFLAGS="$LDFLAGS -lpython$PY_LIB_VER -lc++_shared"
     export LDSHARED="$CXX -shared"
     export CROSS_COMPILE="$ARCH"
     export CROSS_COMPILE_TARGET='yes'
     export _PYTHON_HOST_PLATFORM="android-$ARCH"
 
+    # # Install in host python
+    python setup.py develop
+
+    # Write parse tables
+    python $RECIPE_DIR/write_tables.py
+
     # Build
     python setup.py build
 
     # Rename and move all so files to lib
-    cd build/lib.android-$ARCH-$PY_VER/
+    cd build/lib.android-$ARCH-cpython-310/
```

Comment:
I think this should use the `export PY_LIB_VER_NODOTS=$(echo $PY_LIB_VER | tr -d '.')` like you added in some of the other packages.